docs/actions/user-registration.md

```diff
@@ -0,0 +1,43 @@
+| [Update User Profile](/docs/actions/update-user-profile) | Updates a user's extended profile properties. Use in conjunction with User Registration to capture all data from a registration form in the correct place in the database. |
+| [Send Email](/docs/actions/send-email) | Sends an email to one or multiple recipients. Use to notify a user of successful registration and tell them more about their new account. |
+
+## `Input Parameter Reference`
+
+The parameters unique to this action are listed below. [Review the common parameters for all actions here](/docs/actions/common-parameters).
+
+| Parameter | Description | Supports Tokens | Default | Required |
+| -- | -- | -- | -- | -- |
+| UserName | Select the field that contains the new user's username or use expressions (`EXPR` button) to use a token.  | Yes | `empty string` | No |
+| Register with email | When this option is enabled, the email field selected will also be used as the username for registration. | No | false | No |
+| Password | Select a form field from the dropdown to use as the password for the new user or use expressions (`EXPR` button) to input a custom password or a token. | Yes | Not Selected | No |
+| Generate random password | When this option is enabled, a random password is generated and stored in a context token called `[RegRandomPass]` to be used in subsequent actions. You could use this token in an email action to send the password to the new user. | No | false | No |
+| Email | Select a form field from the dropdown to use as the Email for the new user or use expressions (`EXPR` button) to use a token. | Yes | Not Selected | No |
+| First Name | Select a form field from the dropdown to update the First Name of the user or use expressions (`EXPR` button) to use a token. | Yes | Not Selected | No |
+| Last Name | Select a form field from the dropdown to update the Last Name of the user or use expressions (`EXPR` button) to use a token. | Yes | Not Selected | No |
+| Send standard DNN registration email | When this option is enabled, our module will use the standard registration email as set in DNN. For custom emails, disable this option and add a [Send Email](/docs/actions/send-email) action. | No | false | No |
+| Login if user already exists | When this option is enabled, instead of showing a "user already exists" error, this action will attempt to authenticate the user using the username and password provided. If authentication is successful, then subsequent actions are executed. | No | false | No |
+| Authorize users who register with this form | When **Authorize** is selected, users that successfully register with this form will also be authorized. Selecting **Inherit DNN Settings** will use the standard DNN Setting for Authorizing new users. If you have email verification set up so that the user's account is not active until they follow a link from an email, select **Don't Authorize** and handle authorization upon verification. | No | Not Selected | No |
```

Comment:
```suggestion
| Authorize users who register with this form | When **Authorize** is selected, users that successfully register with this form will also be authorized. Selecting **Inherit DNN Settings** will use the standard DNN Setting for Authorizing new users.  Select **Don't Authorize** if email verification is set to not authorize the user’s account until they follow the email link.  Authorization can be handled upon verification. | No | Not Selected | No |
```

Reply:
This one sounded awkward, but my suggestion sounds awkward as well, so please change any way you see fit or swap back to the original.

Reply:
How about, "Select **Don't Authorize** if authorization is handled later, such as after a user clicks an email verification link." 
This sounds less awkward, and is also better because there are, of course, other reasons. Perhaps user's are being authorized manually by a person.